packages/portal/docs/style/Colours.md

```diff
@@ -30,6 +30,7 @@ Colours are also defined on the [Colours page on Figma](https://www.figma.com/fi
           { name: 'white', hex: '#fff' },
           { name: 'smoke', hex: '#f5f5f5' },
           { name: 'offwhite', hex: '#f1f1ee' },
+          { name: 'lightgrey', hex: '#fbfbfb' },
```

Comment:
There are now two lightgrey colours here. Is that as it should be?

Reply:
Nope, the first should be `lightergrey`. That's updated.